modules/post/windows/gather/credentials/packrat_credentials.rb

```diff
@@ -0,0 +1,273 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  # this associative array defines the artifacts known to PackRat
+  APPLICATION_ARRAY = JSON.parse(
+    File.read(File.join(Msf::Config.data_directory, 'packrat', 'artifacts.json')),
+    symbolize_names: true
+  )
+
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Windows Gather Application Artifacts (PackRat)',
+      'Description' => %q{PackRat gathers artifacts of various categories from a large number of applications.
+       Artifacts include: 12 browsers, 13 chat/IM/IRC applications, 6 email clients, and 1 game.
+       Credentials are then extracted from the artifacts. The use case for this post-exploitation module is
+       to specify the types of
+       artifacts you are interested in, to gather the relevant files depending on your aims.
+       Please refer to the options for a full list of filter categories.},
+      'License' => MSF_LICENSE,
+      'Author' =>
+        [
+          'Daniel Hallsworth', # Leeds Beckett University student
+          'Barwar Salim M', # Leeds Beckett University student
+          'Z. Cliffe Schreuders' # Leeds Beckett University lecturer (http://z.cliffe.schreuders.org)
+        ],
+      'Platform' => %w{win},
+      'SessionTypes' => ['meterpreter']
+      ))
+
+    register_options(
+      [
+        OptRegexp.new('REGEX', [false, 'Match a regular expression', '^password']),
+        OptBool.new('STORE_LOOT', [false, 'Store artifacts into loot database (otherwise, only download)', 'true']),
+        # enumerates the options based on the artifacts that are defined below
+        OptEnum.new('APPCATEGORY', [false, 'Category of applications to gather from', 'All', APPLICATION_ARRAY.map {|x| x[:category]}.uniq.unshift('All')]),
```

Comment:
```suggestion
        OptEnum.new('APP_CATEGORY', [false, 'Category of applications to gather from', 'All', APPLICATION_ARRAY.map {|x| x[:category]}.uniq.unshift('All')]),
``` 
and refactor references